unit_tests/test_get_columns_for_primary_cluster_keys.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from unittest.mock import MagicMock
+from sdcm.utils.nemesis_utils.indexes import get_column_names
+
+
+class TestGetColumnNamesWithCQLTypes(unittest.TestCase):
```

Comment:
I would suggest using pytest instead, as we want to migrate to using it in the near future

Reply:
we already using pytest as test runner for the unit tests and integration tests,

it can run both types of tests, but we can build the same without test classes (and utilize more of pytest features)

Reply:
@timtimb0t 

regardless I would recommend have integration test for this function, that works with actual scylla 
like the one that I pointed to

Reply:
I would like to not introduce anymore unittests cases unless it is needed, I think it is strictly worse solution than using pytest directly